app/mailers/user_mailer.rb

```diff
@@ -210,6 +210,14 @@ def deleted_user_accounts_report(email:, name:, issuers:, data:)
     mail(to: email, subject: t('user_mailer.deleted_accounts_report.subject'))
   end
 
+  def verification_errors_report(email:, name:, issuers:, data:)
+    @name = name
+    @issuers = issuers
+    @data = data
+    attachments['verification_errors.csv'] = data
+    mail(to: email, subject: t('user_mailer.verification_errors_report.subject'))
+  end
+
```

Comment:
this is unused now, let's remove
```suggestion
```

Reply:
The story mentioned emailing the report so I thought saving to s3 was in addition to emailing the report